gitpod-io/website
# Fix typos and capitalize proper nouns

## Description

- Fix typos
- Remove duplicate words
- Capitalize proper nouns

## Related Issue(s)

No related issues.

## How to test

1. Review the changes in the PR diff.
1. Check if I'm changing the correct files.
1. Check my PR title.
1. Check my `release-note`.

## Release Notes

```release-note
Fix typos and capitalize proper nouns
```

## Documentation


No.

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2399"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>
